tests/e2e/test_main.py

```diff
@@ -336,11 +348,18 @@ def test_ignored_files_are_not_copied(
 
     img_uri_str = f"image:extras-e2e:{uuid.uuid4()}"
 
-    result = cli_runner(
-        ["neuro", "image-build", "-f", str(dockerfile_path), ".", img_uri_str]
-    )
+    try:
+        result = cli_runner(
+            ["neuro", "image-build", "-f", str(dockerfile_path), ".", img_uri_str]
+        )
 
-    assert ignored_file_content not in result.stdout
+        assert ignored_file_content not in result.stdout
+    finally:
+        try:
+            cli_runner(["neuro", "image", "rm", img_uri_str])
+        except Exception as e:
+            # Log and ignore exception
+            logger.exception(e)
```

Comment:
let's mute this at all. `logger.exception` will print stacktrace, which, I think, is irrelevant here. So either just `error()` or nothing